pkgs/development/python-modules/dash-bootstrap-components/default.nix

```diff
@@ -28,8 +27,7 @@ buildPythonPackage rec {
   # Tests a additional requirements
   doCheck = false;
 
-  # Circular import
-  # pythonImportsCheck = [ "dash_bootstrap_components" ];
+  pythonImportsCheck = [ "dash_bootstrap_components" ];
```

Comment:
```suggestion
  pythonImportsCheck = [
    "dash_bootstrap_components"
    "dash_bootstrap_components._components"
  ];
```

This catches the issue. Tested locally.

Reply:
thanks for reviewing! I don't think adding the extra import check is necessary, the `dash_bootstrap_components` import on its on was failing without this change, but had been commented out for some reason.